### PR TITLE
Fix missing array code convention at Test files

### PR DESCRIPTION
**Pull request by Monsteel**
Bug/issue #, if applicable: N/A

## Summary

Fix missing array code convention at Test files

## Dependencies

N/A

## Testing

No functional changes.

## Checklist

- [x] Added tests
- [x] Ran the `./bin/test` script and it succeeded
- [x] ~Updated documentation if necessary~ N/A

### CODE REVIEW

File: Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift [hunk 1]
```diff
@@ -179,7 +196,9 @@ class DataAssetManagerTests: XCTestCase {
     
     func testFuzzyLookup() throws {
         var manager = DataAssetManager()
-        let images = ["image.png", "woof~dark.JPG"].compactMap(URL.init(string:))
+        let images = ["image.png",
+                      "woof~dark.JPG",
+        ].compactMap(URL.init(string:))
```
[the remark this answers is not in the document]
Comment (Monsteel) [reply Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift]: @franklinsch  Thank you for your kind comment. 🙇🏻‍♂️
I've checked all the codes related to your comment, I reflected your comment.

I did my best. ☺️

File: Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift [hunk 2]
```diff
@@ -179,7 +199,15 @@ class DataAssetManagerTests: XCTestCase {
...
         var manager = DataAssetManager()
-        let images = ["image.png", "woof~dark.JPG"].compactMap(URL.init(string:))
+        let images = [
+<<<<<<< Updated upstream
+            "image.png",
+            "woof~dark.JPG",
+=======
+          "image.png",
+          "woof~dark.JPG",
+>>>>>>> Stashed changes
```
Comment (franklinsch) [inline Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift:209]: It looks like there is a conflict here

File: Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift [hunk 3]
```diff
@@ -196,12 +224,19 @@ class DataAssetManagerTests: XCTestCase {
...
+          "image@2x.png",
+          "assets/woof.JPG",
+        ].compactMap(URL.init(string:))
         try manager.register(data: images)
         XCTAssertEqual(manager.fuzzyKeyIndex.keys.sorted().map({"\($0)"}), ["image", "woof"])
 
         // Test we include only unique keys in the fuzzy index
-        let imagesWithDuplicates = ["image@2x.png", "assets/woof.JPG", "woof.png"].compactMap(URL.init(string:))
+        let imagesWithDuplicates = [
+          "image@2x.png",
```
Comment (franklinsch) [inline Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift:236]: To be consistent with the rest of the code, we should use a 4-space indent for these

File: Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift (hunk 2 above)
Comment (Monsteel) [reply Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift]: I'm sorry, but I couldn't check it. 😓

File: Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift (hunk 3 above)
Comment (Monsteel) [reply Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift]: Thank you for your kind comment. ☺️

File: Tests/SwiftDocCTests/Infrastructure/DocumentationContextTests.swift [hunk 4]
```diff
@@ -553,8 +553,24 @@ class DocumentationContextTests: XCTestCase {
             .flatMap { $0.variants.map { $0.value.lastPathComponent } }
             .sorted()
         
-        XCTAssertEqual(["figure1.jpg", "figure1.png", "figure1~dark.png", "intro.png", "introposter.png", "introposter2.png", "something@2x.png", "step.png", "titled2up.png", "titled2upCapital.PNG", "with spaces.png", "with spaces@2x.png"], imagesRegistered.sorted())
-    }
+        XCTAssertEqual(
+          [
+            "figure1.jpg",
```
Comment (franklinsch) [inline Tests/SwiftDocCTests/Infrastructure/DocumentationContextTests.swift:558]: This one is still using a two-space indent. I'd expect 4 spaces here for consistency

File: Tests/SwiftDocCTests/Infrastructure/SymbolReferenceTests.swift [hunk 5]
```diff
@@ -211,9 +211,11 @@ class SymbolReferenceTests: XCTestCase {
         // The overloads are sorted and all dupes get a hash suffix.
         XCTAssertEqual(
             context.knownIdentifiers.map { $0.path }.filter { $0.contains("/function") }.sorted(),
-            ["/documentation/OverloadKit/function(_:)-1echv",
-             "/documentation/OverloadKit/function(_:)-4s9wl",
-             "/documentation/OverloadKit/function(_:)-7kt4g"]
+            [
+              "/documentation/OverloadKit/function(_:)-1echv",
```
Comment (franklinsch) [inline Tests/SwiftDocCTests/Infrastructure/SymbolReferenceTests.swift:215]: This one as well

**Review by franklinsch — COMMENTED** [review]
Thanks @Monsteel ! It looks like there are still some cases where we're using two-space indents. Since the goal of this PR is to introduce consistency, I think we should fix those up before merging :)

**Comment by Monsteel** [comment]
@franklinsch I'm sorry I couldn't look at all the files with interest.
Among the files I changed, I checked and corrected the incorrect part of space indent convention.

If there's anything I missed,
Please let me know. I'll revise it.

Thank you for your kind confirmation.

**Comment by Monsteel** [comment]
@franklinsch Please check..!😓

**Comment by ethan-kusters** [comment]
Hi @Monsteel! This looks good-to-go from my perspective. Thank you for your work here!

If you could resolve the remaining conflicts, I'll re-run CI and merge.

**Comment by Monsteel** [comment]
Hello @ethan-kusters ! Thank you so much. 🤗
I solved the remaining conflicts.🙋🏻‍♂️ Please check. 🙏

**Comment by ethan-kusters** [comment]
@swift-ci please test

**Comment by ethan-kusters** [comment]
@swift-ci please test

**Comment by Monsteel** [comment]
@ethan-kusters Thanks, But I don't understand swiftTest result. please tell me about test result ?😓

**Comment by ethan-kusters** [comment]
@Monsteel could you manually rebase on `main` and force-push your changes? It looks like just merging main in the GitHub UI is not picking up the latest changes.

**Comment by Monsteel** [comment]
@ethan-kusters I did manually rebase on main and force-push my changes🙋🏻‍♂️ Thanks for your hard work! 🙇🏻‍♂️

**Comment by ethan-kusters** [comment]
@swift-ci please test

**Review by ethan-kusters — APPROVED** [review]
Thanks @Monsteel!

**Comment by Monsteel** [comment]
Thank you, @ethan-kusters, @franklinsch !

Through this opportunity, **I felt the kindness of Apple developer,  I experienced small collaboration.** so I'm happy.🤩
I hope you have a happy coding. 🙏

Thanks!

**Pull request merged**